content/_data/changelogs/weekly.yml

```
@@ -14565,6 +14565,74 @@
   # pull: 6313 (PR title: Clarify that the changelog entry should be in the imperative mood)
   # pull: 6314 (PR title: Bump script-security from 1138.v8e727069a_025 to 1140.vf967fb_efa_55a_)
 
+  - version: '2.338'
+    date: 2022-03-06
+    changes:
+      - type: rfe
+        category: developer
+        pull: 6317
+        authors:
+          - glasswalk3r
+          - 'TODO: alceu.freitas (alceu.freitas@semantix.com.br)'
+        pr_title: Refactor `translation-tool.pl`
+        message: |-
+          Developer: Add a <code>--debug</code> option to the translation tool to help in troubleshooting.
+           Developer: Improve the readability of the translation tool command-line interface (CLI).
+      - type: TODO
+        category: developer
+        pull: 6329
+        authors:
+          - dependabot[bot]
+        pr_title: Bump Remoting from 4.12 to 4.13
+        message: |-
+          Allow agent reconnects on Java 11 and websocket ([JENKINS-66446](https://issues.jenkins.io/browse/JENKINS-66446) - [PR-514](https://github.com/jenkinsci/remoting/pull/514) ). Prevent infinite loop in case of closed SSL connection ([JENKINS-67928](https://issues.jenkins.io/browse/JENKINS-67928) - [PR-513](https://github.com/jenkinsci/remoting/pull/513) ) Upgrade from  Remoting 4.12 to [4.13](https://github.com/jenkinsci/remoting/releases/tag/remoting-4.13).
```

```suggestion
          Developer: Allow agent reconnects on Java 11 and WebSocket.
          Prevent infinite loop in case of a closed SSL connection.
          Upgrade from  Remoting 4.12 to 4.13.
```

To be done with the help of a separate PR after the release:
- 'developer' category entry needs to be placed towards the end.
- All the related links mentioned in the message need to be placed under the 'references' section, like this:
```
references:
          - pull: 6317
          - url: https://issues.jenkins.io/browse/JENKINS-66446
            title: JENKINS-66446
          - url: https://github.com/jenkinsci/remoting/pull/514
            title: PR-514
          - url: https://issues.jenkins.io/browse/JENKINS-67928
            title: JENKINS-67928
          - url: https://github.com/jenkinsci/remoting/pull/513
          - title: PR-513
          - url: https://github.com/jenkinsci/remoting/releases/tag/remoting-4.12
          - title: Remoting 4.12 changelog
          - url: https://github.com/jenkinsci/remoting/releases/tag/remoting-4.13
          - title: Remoting 4.13 changelog
```

---

This isn't a developer category. This is for users